src/Cody.UI/ViewModels/EditCodeViewModel.cs

```diff
@@ -21,6 +24,9 @@ public EditCodeViewModel(IEnumerable<Model> models, string selectedModelId, stri
             Models = modelsSource;
 
             SelectedModel = collection.FirstOrDefault(x => x.Id == selectedModelId);
```

Comment:
wow. this looks like _a very nice feature of cs_ 😅 

I would expect `FirstOrDefault` to take two params - the condition to fulfil and the default value to return in the condition has not been met by any element. 

but... in C#, the default means the default value for the type that - for reference type - is `null` 😮 

do I understand correclty? don't we need any fallback here? smth like `?? collection.FirstOrDefault()`?